pyproject.toml

```diff
@@ -1,33 +1,19 @@
 [project]
 name = "bblocks"
 version = "1.4.3"
-description = "A package with tools to download and analyse international development data. These tools are meant to be the building blocks of further analysis."
+description = "Building blocks for international development data."
 authors = [
-    {name = "The ONE Campaign",email = "data@one.org"}
+    {name = "ONE Campaign"},
+    {name = "Luca Picci",email = "lpicci96@gmail.com"},
```

Comment:
should be your one email?

Reply:
maybe but all my github work is tied to my personal gmail and it ensures continuity if the one email becomes deactivated in the future. Hopefully this tool will outlast us.